java/src/jmri/jmrix/AbstractConnectionConfig.java

```diff
@@ -229,15 +230,15 @@ protected int addStandardDetails(PortAdapter adapter, boolean incAdvanced, int i
             cL.gridy = i;
             gbLayout.setConstraints(systemPrefixLabel, cL);
             gbLayout.setConstraints(systemPrefixField, cR);
-	    systemPrefixLabel.setLabelFor(systemPrefixField);
+            systemPrefixLabel.setLabelFor(systemPrefixField);
             _details.add(systemPrefixLabel);
             _details.add(systemPrefixField);
             i++;
             cR.gridy = i;
             cL.gridy = i;
             gbLayout.setConstraints(connectionNameLabel, cL);
             gbLayout.setConstraints(connectionNameField, cR);
-	    connectionNameLabel.setLabelFor(connectionNameField);
+	        connectionNameLabel.setLabelFor(connectionNameField);
```

Comment:
The TAB character is still there so the spacing gets wrong.

Reply:
Thanks, done. Hard to spot on the screen. Checked that I have the whitespace style prefs set to spaces in Ij.